### ISSUE TITLE
Contextual type 'AnyObject' cannot be used with dictionary literal

### ISSUE THREAD

**Issue by EkambaramE** [xyclos/SwiftChat#1]
```
func textFieldShouldReturn(textField: UITextField!) -> Bool {
    textField.endEditing(true)

   self.firebase.childByAutoId().setValue(["name": self.name, "text": msgInput!.text]) ///<--- From this line i'm getting error

    textField.text = ""
    return false
}
```

**Comment by xyclos** [comment]
Thanks for posting this issue. I'll see if I can address this soon.

**Comment by xyclos** [comment]
This is fixed in [a66746e](https://github.com/xyclos/SwiftChat/commit/a66746e19d6848bfe5f126985d4ae85b9a522b9a)

**Comment by sametgolgeci** [comment]
self.firebase.childByAutoId().setValue(["name": self.name as! anyObject, "text": msgInput!.text as! anyObject])
